core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -429,6 +434,27 @@ private void setLocationOnce(String val, String qual, LocationType lt) {
     location = new Location(val, qual, lt);
   }
 
+  static Iterable<TabletMetadata> convert(BatchScanner input, EnumSet<ColumnType> fetchedColumns,
+      boolean buildKeyValueMap) {
+
+    IteratorSetting iterSetting = new IteratorSetting(100, WholeRowIterator.class);
+    input.addScanIterator(iterSetting);
```

Comment:
It seems like a bad idea to put these inside this method, as they mutate BatchScanner, which could be re-used by the caller. This requires a significant amount of coordination between the caller and this method's implementation, such that it probably doesn't make sense to have this method. Since there's only one caller, it could be inline'd... or at least made local and private for the one caller.

Reply:
> It seems like a bad idea to put these inside this method, as they mutate BatchScanner, which could be re-used by the caller. 

Yeah that was awful.  I was following the pattern of another existing method, that had the same bad pattern.  I inlined both methods in 887a91b